lib/src/replicache.dart

```diff
@@ -611,3 +631,21 @@ class _MutateResult<R> {
   final String ref;
   _MutateResult(this.result, this.ref);
 }
+
+final _inTxKey = {};
```

Comment:
[cbb?] Can you make this a constant local inside `_guardTransaction` and get the same effect?

Reply:
No. This is used as the key into the zones map and it needs to be shared between calls to `_guardTransaction` and we want it to be unguessable (`const {} == const {}`).